dse-68/Dockerfile.jdk11

```diff
@@ -0,0 +1,180 @@
+# Copyright DataStax, Inc, 2017
```

Comment:
All of the Docker build files were generated from the `docker-images` repo with the changes from this PR:
https://github.com/riptano/docker-images/pull/87
There are some slight modifications to the build files as that PR will add a released Management API bundle to the DSE image, where the files in this PR will compile the latest Management API bundles and add them to the images built in this repo.